_studio/shared/src/mfx_vpp_vaapi.cpp

```diff
@@ -1056,7 +1056,16 @@ mfxStatus VAAPIVideoProcessing::Execute(mfxExecuteParams *pParams)
     output_region.width  = outInfo->CropW;
     m_pipelineParam[0].output_region = &output_region;
 
-    m_pipelineParam[0].output_background_color = 0xff000000; // black for ARGB
+    m_pipelineParam[0].output_background_color = 0xff000000;
+
+    // It looks like only BXT supports this at the moment
+    if (MFX_HW_APL == hwType &&
```

Comment:
@onabiull : do we have an uAPI in mediasdk to setup background color? do we have an uAPI to switch off background color entirely?

Reply:
@dvrogozh, 
Answers:
(1): no MSDK do not have uAPI for setup background color.
(2): MSDK do not have uAPI to switch off background color entirely

Reply:
@dvrogozh 
If you don't have concern any more I would like to merge it.